adapters/context_adapter.py

```diff
@@ -188,17 +190,19 @@ def _verify_in_node_operation(self):
         if self.type != NODE_INSTANCE:
             self._ctx.task.abort(
                 'ctx.node/ctx.instance can only be used in a {0} context but '
-                'used in a {1} context.'.format(NODE_INSTANCE, self.type))
+                'used in a {1} context.'.format(NODE_INSTANCE, self.type)
+            )
 
     def _verify_in_relationship_operation(self):
         if self.type != RELATIONSHIP_INSTANCE:
             self._ctx.task.abort(
                 'ctx.source/ctx.target can only be used in a {0} context but '
                 'used in a {1} context.'.format(RELATIONSHIP_INSTANCE,
-                                                self.type))
+                                                self.type)
+            )
 
 
-class _Blueprint(object):
+class BlueprintAdapter(object):
```

Comment:
I specifically removed the _. Why do you think they cannot be public? These could be potentially useful building blocks for constructing adapters programmatically for other uses outside of the executor wrapper. I don't see a reason to restrict them, as they are general-purpose adapters.